java/feathub-connectors/flink-connectors/flink-connector-redis/src/main/java/com/alibaba/feathub/flink/connectors/redis/sink/RedisSinkConfigs.java

```diff
@@ -0,0 +1,80 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.feathub.flink.connectors.redis.sink;
+
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ConfigOptions;
+
+/** Configurations used by Redis sink. */
+public class RedisSinkConfigs {
+
+    static final ConfigOption<String> HOST =
+            ConfigOptions.key("host")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription("The host of the Redis instance to connect.");
+
+    static final ConfigOption<Integer> PORT =
+            ConfigOptions.key("port")
+                    .intType()
+                    .defaultValue(6379)
+                    .withDescription("The port of the Redis instance to connect.");
+
+    static final ConfigOption<String> PASSWORD =
+            ConfigOptions.key("password")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription("The password of the Redis instance to connect.");
+
+    static final ConfigOption<Integer> DB_NUM =
+            ConfigOptions.key("dbNum")
+                    .intType()
+                    .defaultValue(0)
+                    .withDescription("The No. of the Redis database to connect.");
+
+    static final ConfigOption<String> TABLE_NAME =
+            ConfigOptions.key("tableName")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "A name that uniquely identifies the input table. "
+                                    + "Tables with different tableName can have the same key.");
+
+    static final ConfigOption<String> KEY_FIELDS =
+            ConfigOptions.key("keyFields")
+                    .stringType()
+                    .defaultValue("")
+                    .withDescription(
+                            "The names of the fields that are selected as keys. "
+                                    + "Names are separated by semicolons(;).");
+
+    static final ConfigOption<String> TIMESTAMP_FIELD =
+            ConfigOptions.key("timestampField")
+                    .stringType()
+                    .defaultValue("")
+                    .withDescription(
+                            "The name of the field representing the timestamp of the row. "
+                                    + "Records with the same key but smaller timestamp value "
+                                    + "would not be written to Redis. ");
+
+    static final ConfigOption<String> TIMESTAMP_FORMAT =
+            ConfigOptions.key("timestampFormat")
+                    .stringType()
+                    .defaultValue("")
+                    .withDescription(
+                            "The format of the timestamp field values if the values are Strings.");
```

Comment:
Why do we limit the format to only apply to string values? Would it be better to keep the format semantics consistent with FeatHub's public API?